packages/components/dialog/src/dialog.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <teleport to="body" :disabled="!appendToBody">
+  <teleport :to="appendTo" :disabled="!appendToBody && !appendTo">
```

Comment:
The value of disabled is `true` before modification and then it is `false`. You should use `||`.

Reply:
Isn't it `undefined` by default? https://github.com/element-plus/element-plus/blob/e32cef1d1b269ea16752b0ac15c5bab41fae9038/packages/components/dialog/src/dialog.ts#L15

Reply:
Isn't it `false` by default? https://github.com/element-plus/element-plus/blob/e32cef1d1b269ea16752b0ac15c5bab41fae9038/packages/components/dialog/src/dialog.ts#L15

Reply:
I found a fix, seems like tests are passing successfully now